nyc_taxis/operations/default.json

```diff
@@ -25,54 +25,6 @@
       "retry-until-success": true,
       "include-in-reporting": false
     },
-    {
```

Comment:
Looks good. It's better that this is moved to a separate file. We'll create a statement in this workload that states that `default.json` in operations and test_procedures shouldn't be updated directly but rather in separate files (unless to fix breaking changes or other exceptions)